<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QOperateur</class>
 <widget class="QWidget" name="QOperateur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="grpBox_opera">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>271</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Operator :</string>
   </property>
   <widget class="QLabel" name="lbl_AR">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>AR</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_D1R">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>D1R</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_SL">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SL</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_D2R">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>D2R</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_RR">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>RR</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_ARvel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>AR. Vel.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_lvlAR">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Lvl. Vel.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_detune">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Fine</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_coarse">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Coarse</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_multiple">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Mult.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_lvldph">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Lvl. Dph.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_lvlcurb">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Lvl. Curb.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_rtdph">
    <property name="geometry">
     <rect>
      <x>112</x>
      <y>170</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Rt. Dph.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_adjTL">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Adj. Vol</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pshBut_carrier">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>The operator output mode</string>
    </property>
    <property name="text">
     <string>Car.</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_carrmod">
    <property name="geometry">
     <rect>
      <x>158</x>
      <y>120</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Car./Mod.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_volume">
    <property name="geometry">
     <rect>
      <x>204</x>
      <y>120</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Volume</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbBox_lvlcurb">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>The response curb of the output level to the note height</string>
    </property>
    <item>
     <property name="text">
      <string>-Lin</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>+Lin</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>-Exp</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>+Exp</string>
     </property>
    </item>
   </widget>
   <widget class="QEnveloppe" name="label_env">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>26</y>
      <width>118</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="hzSlider_volume">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>21</width>
      <height>61</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>The output level of the operator</string>
    </property>
    <property name="maximum">
     <number>127</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QButton" name="but_vellvl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>The influence of key velocity on the output level</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="minValue" stdset="0">
     <number>0</number>
    </property>
    <property name="maxValue" stdset="0">
     <number>31</number>
    </property>
   </widget>
   <widget class="QButton" name="but_velAR">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>The influence of key velocity on attack rate</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="minValue" stdset="0">
     <number>0</number>
    </property>
    <property name="maxValue" stdset="0">
     <number>3</number>
    </property>
   </widget>
   <widget class="QButton" name="but_AR">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>The attack rate of the enveloppe</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="minValue" stdset="0">
     <number>0</number>
    </property>
    <property name="maxValue" stdset="0">
     <number>31</number>
    </property>
   </widget>
   <widget class="QButton" name="but_D1R">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>The decay 1 rate of the enveloppe</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="minValue" stdset="0">
     <number>0</number>
    </property>
    <property name="maxValue" stdset="0">
     <number>31</number>
    </property>
   </widget>
   <widget class="QButton" name="but_SL">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>The sustain level of the enveloppe</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="minValue" stdset="0">
     <number>0</number>
    </property>
    <property name="maxValue" stdset="0">
     <number>15</number>
    </property>
   </widget>
   <widget class="QButton" name="but_D2R">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>The decay 2 rate of the enveloppe</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="minValue" stdset="0">
     <number>0</number>
    </property>
    <property name="maxValue" stdset="0">
     <number>31</number>
    </property>
   </widget>
   <widget class="QButton" name="but_RR">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>The release rate of the enveloppe</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="minValue" stdset="0">
     <number>0</number>
    </property>
    <property name="maxValue" stdset="0">
     <number>15</number>
    </property>
   </widget>
   <widget class="QButton" name="but_coarse">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>The coarse detune level</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="minValue" stdset="0">
     <number>0</number>
    </property>
    <property name="maxValue" stdset="0">
     <number>3</number>
    </property>
   </widget>
   <widget class="QButton" name="but_mult">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="minValue" stdset="0">
     <number>0</number>
    </property>
    <property name="maxValue" stdset="0">
     <number>15</number>
    </property>
   </widget>
   <widget class="QButton" name="but_fine">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>The fine detune level</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="minValue" stdset="0">
     <number>-4</number>
    </property>
    <property name="maxValue" stdset="0">
     <number>3</number>
    </property>
   </widget>
   <widget class="QButton" name="but_lvldph">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>The influence of the note height on the output level</string>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="minValue" stdset="0">
     <number>0</number>
    </property>
    <property name="maxValue" stdset="0">
     <number>15</number>
    </property>
   </widget>
   <widget class="QButton" name="but_rtdph">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>The influence of the note height on the enveloppe rates</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="minValue" stdset="0">
     <number>0</number>
    </property>
    <property name="maxValue" stdset="0">
     <number>3</number>
    </property>
   </widget>
   <widget class="QButton" name="but_adjTL">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Fine adjustement for the output level</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="minValue" stdset="0">
     <number>0</number>
    </property>
    <property name="maxValue" stdset="0">
     <number>15</number>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QButton</class>
   <extends>QPushButton</extends>
   <header>interface/qbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QEnveloppe</class>
   <extends>QLabel</extends>
   <header>interface/qenveloppe.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
